proposed/0006-Launcher.md

```diff
@@ -0,0 +1,204 @@
+- Feature Name: SAFE-Launcher
```

Comment:
To disambiguate it from the "Launcher-Simplified" RFC, this should be renamed to something like "Launcher as a service/daemon".